### ISSUE TITLE
Fish completion

### ISSUE THREAD

**Issue by beeaniebee** [tadfisher/pass-otp#118]
Just wondering if there's any way that fish shell completion could be implemented? It would really help a lot!

**Comment by roddhjav** [comment]
`pass` does not support fish shell completion for extension (yet). So the support should be implemented in pass first.

**Comment by beeaniebee** [comment]
No? Fish shell completion is working fine in pass.. it's in src/completion/fish_shell.completion in the source code iirc

**Comment by roddhjav** [comment]
Yes, but this completion file is not able to check for extension completion file and run them as the bash completion file does.

**Comment by beeaniebee** [comment]
Ok.. I don't exactly understand all of that but ok

**Comment by harmtemolder** [comment]
I've found a way to `source` the original `pass` completions and add the ones for `pass otp`. Working on getting that production-ready in pull request #151
